src/Api/Request/ExportUsersBySegmentRequest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Lingoda\BrazeBundle\Api\Request;
+
+use Lingoda\BrazeBundle\Api\Object\SegmentId;
+use Lingoda\BrazeBundle\Api\Object\Traits\OptionsTrait;
+use Lingoda\BrazeBundle\Validator\Validation;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Webmozart\Assert\Assert;
+
+class ExportUsersBySegmentRequest
+{
+    use OptionsTrait {
+        setOptions as setResolvedOptions;
+    }
+
+    protected function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefined([
+            'segment_id',
+            'callback_endpoint',
+            'fields_to_export',
+        ]);
+
+        $resolver
+            ->setAllowedTypes('segment_id', SegmentId::class)
+            ->setAllowedTypes('callback_endpoint', 'string')
+            ->setAllowedTypes('fields_to_export', 'string[]')
+        ;
+
+        $resolver
+            ->setAllowedValues('segment_id', Validation::createIsValidCallback(
```

Comment:
Is it intentional that you have 'isValidCallback' validation rule on the 'segment_id'?